.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 180
```

Comment:
We should consider putting a tighter bound for stale PRs, maybe 3 months? You can specify PR specific settings in a `pulls` section, see https://github.com/probot/stale#usage for details

Reply:
updated!